packages/flutter/lib/src/rendering/semantics.dart

```diff
@@ -312,6 +331,16 @@ class SemanticsNode extends AbstractNode {
     }
   }
 
+  Set<SemanticsTag> _tags = new Set<SemanticsTag>();
+
+  /// Tag the [SemanticsNode] with [tag].
+  void addTag(SemanticsTag tag) {
```

Comment:
It sounds like a bug to me if one RenderObject's annotator decides checked should be true on a given SemanticsNode and another RenderObject's annotator decides checked should be false on the same SemanticsNode. What's the use case?

Similarly, removing a tag seems like an obscure use case (most likely indicating a bug). Do you have anything in mind where that's actually necessary? I'd prefer to keep the API as simple as possible to avoid that people think they have to manually manage the removal of tags.